components/TxForm.tsx

```diff
@@ -4,6 +4,29 @@ import { FormControl, FormLabel } from "@chakra-ui/form-control";
 import { Input, InputGroup, InputRightElement } from "@chakra-ui/input";
 import { Box, Flex } from "@chakra-ui/layout";
 import { Select } from "@chakra-ui/select";
+import { keccak256 } from "@railgun-community/engine";
+import {
+  getShieldPrivateKeySignatureMessage,
+  gasEstimateForShield,
+  getRailgunSmartWalletContractForNetwork,
+  populateShield,
+} from "@railgun-community/quickstart";
+import {
+  RailgunERC20AmountRecipient,
+  NetworkName,
+  TransactionGasDetailsSerialized,
+  EVMGasType,
+  NETWORK_CONFIG,
+  deserializeTransaction,
+} from "@railgun-community/shared-models";
+import { BigNumber, Signer, ethers, constants } from "ethers";
```

Comment:
I like to avoid importing the full `ethers` object directly and instead import the specific things you need, in this particular case `import type { TransactionRequest } from '@ethersproject/providers'`. Similarly, `import { MaxUint256 } from '@ethersproject/constants'` can be used instead of importing all constants

 The reason is because I don't know how good tree shaking is and don't want to accidentally leave in the full ethers bundle. _In theory_ this shouldn't matter, but just wanted to mention it in case you have any thoughts here. Not saying we must change this, feel free to resolve this comment yourself